src/main/java/Mikey.java

```diff
@@ -0,0 +1,295 @@
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.StringJoiner;
+import java.util.Arrays;
+
+public class Mikey {
+
+    public static ArrayList<Task> tasks = new ArrayList<Task>();
+
+    public static Task newTodo(String taskName, int completion) {
+        Task newTask = null;
+        newTask = new Todo(taskName);
+        if(completion == 1) {
+            newTask.isDone = true;
+        } else {
+            newTask.isDone = false;
+        }
+        tasks.add(newTask);
+        return newTask;
+    }
+
+    public static Task newDeadline(String taskName, String dateDue, int completion) {
+        Task newTask = null;
+        newTask = new Deadline(taskName, dateDue);
+        if(completion == 1) {
+            newTask.isDone = true;
+        } else {
+            newTask.isDone = false;
+        }
+        tasks.add(newTask);
+        return newTask;
+    }
+
+    public static Task newEvent(String taskName, String timeOfEvent, int completion) {
+        Task newTask = null;
+        newTask = new Event(taskName, timeOfEvent);
+        if(completion == 1) {
+            newTask.isDone = true;
+        } else {
+            newTask.isDone = false;
+        }
+        tasks.add(newTask);
+        return newTask;
+    }
+    public static void printTask(int taskNumber) {
+        System.out.println("[" + tasks.get(taskNumber).getTaskType() + "]" + "["
+                + tasks.get(taskNumber).getStatusIcon() + "] " +  tasks.get(taskNumber).getName() + " "
+                    + tasks.get(taskNumber).getDate());
+        System.out.println();
+    }
+
+    public static void addTaskMessage() {
+        System.out.println("Now yous got " + tasks.size() + " thing(s) ta do in ya list");
+        System.out.println("Anythin else ya need me ta help ya with?");
+        System.out.println();
+    }
+
+    public static void deleteTask(int taskNumber) {
+        System.out.println("I've gotcha mate, removed [" + tasks.get(taskNumber).getTaskType() + "]" + "["
+            + tasks.get(taskNumber).getStatusIcon() + "]" + tasks.get(taskNumber).getName() + " "
+                + tasks.get(taskNumber).getDate());
+        tasks.remove(taskNumber);
+    }
+
+    public static void saveToFile() throws java.io.IOException {
+        StringJoiner taskFormatter = new StringJoiner(System.lineSeparator());
+        for (Task t: tasks) {
+            taskFormatter.add(t.taskStringFormat());
+        }
+        int count = 0;
+        int maxTries = 1;
+        while (true) {
+            try {
+                FileWriter listWrite = new FileWriter("./data/Mikey.txt");
+                listWrite.write(taskFormatter.toString());
+                listWrite.close();
+                break;
+            } catch (FileNotFoundException e) {
+                fileNotFound(taskFormatter.toString());
+            }
+        }
+    }
+
+    public static void readFromFile() throws java.io.IOException{
+        try {
```

Comment:
This method is very long >30 lines. Consider refactoring some parts of this function to other functions and call them. [See here](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods)